### PR TITLE
1562: The method 'CheckRun#findComment' don't need the comment list as parameter

### PR DESCRIPTION
**Pull request by zhaosongzs**
remove parameter 'comments' in CheckRun#findComment.

remove parameter 'activeReviews' in many method interface.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-1562](https://bugs.openjdk.org/browse/SKARA-1562): The method 'CheckRun#findComment' don't need the comment list as parameter

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara pull/1375/head:pull/1375` \
`$ git checkout pull/1375`

Update a local copy of the PR: \
`$ git checkout pull/1375` \
`$ git pull https://git.openjdk.org/skara pull/1375/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1375`

View PR using the GUI difftool: \
`$ git pr show -t 1375`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1375.diff">https://git.openjdk.org/skara/pull/1375.diff</a>

</details>

### CODE REVIEW

**Comment by erikj79** [comment]
Looks like activeReviews could be eliminated the same way.

Also please note that you need to describe your change in the PR body. The bot doesn't accept an empty body as you can see from the error message here.

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ private void checkStatus() {
             var integrationBlockers = botSpecificIntegrationBlockers();
 
             // Calculate and update the status message if needed
-            var statusMessage = getStatusMessage(comments, activeReviews, visitor, additionalErrors,
+            var statusMessage = getStatusMessage(visitor, additionalErrors,
                                                 additionalProgresses, integrationBlockers, isCleanBackport);
```
Comment (erikj79) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java:1074]: Could you fixup the linebreak here.

Comment (zhaosongzs) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java:1074]: Sure, thanks for your advice!

**Review by erikj79 — APPROVED** [review]

**Pull request closed without merging**